cypress/integration/accessibility_Heading_spec.js

```diff
@@ -5,6 +5,15 @@ describe('Heading Accessibility check', () => {
   });
 
   it('Tests accessibility on the Heading page', () => {
+    // Heading docs showcase all sizes, breaking the order
```

Comment:
Rather than disabling the rule entirely, can we add test ids to the incorrect sections and exclude those? (What I was trying on that other PR and failed at — maybe you can get it to work here?)

Reply:
Now that we've proven disabling sections of the page, perhaps we can use that here rather than disabling the rule?

Reply:
I think in this case it still makes sense to disable the rule for the whole page, the same way we do for PageHeader. Any example on the Heading page will break the heading order, so rather than disable it for every single example, current and future, I think it's a better developer experience to know that's being taken care of at the page level, and disabling it has no adverse affect on the accessibility of our product since heading-order isn't something we can control

Reply:
Plus when we disable using the class name, we are disabling every accessibility check. By disabling just the heading-order test on the page, the rest of our accessibility tests will run on each example (contrast, link names, etc.)

Reply:
Good points! 👍